internal/clients/ec2/ec2_client.go

```diff
@@ -99,3 +99,17 @@ func (c *Client) CreateEC2ClientFromConfig(crd *stsTypes.Credentials) (*Client,
 
 	return newClient, nil
 }
+
+func (c *Client) ListInstanceTypes() ([]types.InstanceTypeInfo, error) {
+	log.Trace().Msgf("Listing AWS EC2 instance types")
```

Comment:
Use `Msg` when not using formatting:

```suggestion
	log.Trace().Msg("Listing AWS EC2 instance types")
```